pr2_gazebo_plugins/src/gazebo_ros_power_monitor.cpp

```diff
@@ -42,106 +42,167 @@ using namespace std;
 namespace gazebo {
 
 GazeboRosPowerMonitor::GazeboRosPowerMonitor()
-{
-    // model_ = dynamic_cast<Model*>(parent);
-    // if (!model_)
-    //     gzthrow("GazeboRosPowerMonitor controller requires a Model as its parent");
-
-    // // Initialize parameters
-    // Param::Begin(&parameters);
-    // robot_namespace_param_   = new ParamT<string>("robotNamespace",     "/",           0);
```

Comment:
Here any parameters should have been read but no parameters actually read from urdf and all the parameters are zero which can cause zero division error and therefore this plugin publishes wrong trivial data.